backend/src/lastwill/lastwill.service.ts

```diff
@@ -40,4 +54,89 @@ export class LastWillService {
 
     return await this.lastwillDbService.createOne(createLastWillDto, userId)
   }
+
+  includesFinancialInheritance(lastWill: LastWill): boolean {
+    if (lastWill.financialAssets.length === 0) return false
+    for (const heir of lastWill.heirs) {
+      if (heir.percentage) {
+        return true
+      }
```

Comment:
```suggestion
      if (heir.percentage) return true
```

Guard claus onliners better readable